hack/ci/cloud-init/controller.yaml.tpl

```diff
@@ -34,8 +34,10 @@
     # Glance
     ENABLED_SERVICES+=,g-api
 
-    # Neutron
-    ENABLED_SERVICES+=,neutron-api,neutron-agent,neutron-dhcp,neutron-l3
+    # Octavia-Neutron
+    Q_AGENT=linuxbridge
```

Comment:
what's the reason to make it bridge instead of ovs?

Reply:
Thanks, removed it.